Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	2	Reproducibility
	1	STY_plotting
	1	SetRank_analysis
	1	all
	1	create_STY_eSet
	1	filter_eSet
	1	heatmap
	1	load_prepare_STY_files
	1	pca
	1	prepare_shiny_inputs
	1	setRank_collection_create
	1	topTables
	2	volcano_plot
	1	waterfall
	16

[Tue Dec 11 10:57:57 2018]
rule load_prepare_STY_files:
    input: /Users/Mahmoud.Hallal/Desktop/PhD/cell_lines_data/CML_samples_12.2018/pipeline_data.csv, /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/Scripts/cell_lines_analysis_STYfiles_INC1_INC2_sd.R
    output: /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/eSet_data_test1_CML.csv, /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/batches_all_data_CML.Rda, /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/batches_all_phospho_data_CML.Rda
    jobid: 5
    wildcards: CWD=/Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline, new_file=CML_0.01P_0.05FDR, cline=CML

[Tue Dec 11 10:57:58 2018]
Error in rule load_prepare_STY_files:
    jobid: 5
    output: /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/eSet_data_test1_CML.csv, /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/batches_all_data_CML.Rda, /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/results/CML_0.01P_0.05FDR/batches_all_phospho_data_CML.Rda

RuleException:
CalledProcessError in line 72 of /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/snakemake/Snakefile:
Command ' set -euo pipefail;   Rscript /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/Scripts/cell_lines_analysis_STYfiles_INC1_INC2_sd.R ' returned non-zero exit status 1.
  File "/Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/snakemake/Snakefile", line 72, in __rule_load_prepare_STY_files
  File "/Users/Mahmoud.Hallal/miniconda3/envs/snakemake_General/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/Mahmoud.Hallal/Desktop/PhD/Generalized_pipeline/snakemake/.snakemake/log/2018-12-11T105756.640585.snakemake.log
